python/hail/utils/misc.py

```diff
@@ -120,6 +120,14 @@ def new_local_temp_dir(suffix=None, prefix=None, dir=None):
     atexit.register(shutil.rmtree, local_temp_dir)
     return local_temp_dir
 
+
+def new_local_temp_file(filename="temp"):
+    local_temp_dir = new_local_temp_dir()
+    path = local_temp_dir + "/" + filename
+    uri = local_path_uri(path)
+    return path, uri
```

Comment:
This shouldn't return a uri.  (I think you ignore it most places its called, anyway.)

Reply:
I use the both the path and uri everywhere it is called. That’s why I had the name new_local_temp_path_uri before.

Reply:
I changed it to just return the local `path`, and then I compute the `uri` from that externally